src/main/java/bio/terra/service/filedata/google/firestore/FireStoreDao.java

```diff
@@ -165,7 +176,15 @@ public void snapshotCompute(Snapshot snapshot) throws InterruptedException {
             // We batch the updates to firestore by collecting updated entries into this list,
             // and when we get enough, writing them out.
             List<FireStoreDirectoryEntry> updateBatch = new ArrayList<>();
+
+            String retrieveTimer = performanceLogger.timerStart();
             computeDirectory(firestore, snapshotId, topDir, updateBatch);
+            performanceLogger.timerEndAndLog(
+                retrieveTimer,
+                snapshotId, // not a flight, so no job id
```

Comment:
is there no flight/jobId?

Reply:
there didn't seem to be for this method.  I noticed that we do similar things in other calls timerEndAndLog.  I wonder if we really want that second parameter to be flightId or more generally ID